job-dsls/jobs/DailyBuild_pipeline.groovy

```diff
@@ -0,0 +1,613 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def javadk=Constants.JDK_VERSION
+def mvnVersion="kie-maven-3.5.2"
+def javaToolEnv="KIE_JDK1_8"
+def mvnToolEnv="KIE_MAVEN_3_5_2"
+def mvnHome="${mvnToolEnv}_HOME"
+def kieVersion=Constants.KIE_PREFIX
+def baseBranch=Constants.BRANCH
+def organization=Constants.GITHUB_ORG_UNIT
+def deployDir="deploy-dir"
+def m2Dir="\$HOME/.m2/repository"
+
+String EAP7_DOWNLOAD_URL = "http://download.devel.eng.brq.redhat.com/released/JBoss-middleware/eap7/7.2.0/jboss-eap-7.2.0.zip"
```

Comment:
```suggestion
String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.2.0/jboss-eap-7.2.0.zip"
```

Reply:
I think this is the last change pending.

Reply:
DONE